manifest.yaml

```diff
@@ -42,6 +42,15 @@ arguments:
           - kafka
           - python
     description: Any valid built-in service activities can be supplied here and they will be provided. The `service` flag must be set to true. Valid values for this are `http`, `grpc`, `temporal`, `python` and `kafka`.
+  aws.account:
+    schema:
+      type: string
+    description: AWS account ID to use for this repository
```

Comment:
suggestion: this should be a box value instead of a manifest value, as it's unlikely you'll have a different value for this across a given org. For example:

https://github.com/getoutreach/stencil-golang/blob/c82a9c2cb59eaba3b9c375d33c5cd76af75a66af/templates/go.mod.tpl#L3

Reply:
Fixed